From c4dd6fe3c2853bf26a97e3c4dcffd82b03dd52a1 Mon Sep 17 00:00:00 2001
From: AdarshGrewal <adarshgrewal@gmail.com>
Date: Tue, 25 Mar 2025 00:24:09 +0530
Subject: [PATCH 27/28] fixup! SystemUI: Implement Reverse-engineered
 Smartspace

---
 .../systemui/smartspace/KeyguardMediaViewController.java      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/google/android/systemui/smartspace/KeyguardMediaViewController.java b/packages/SystemUI/src/com/google/android/systemui/smartspace/KeyguardMediaViewController.java
index e39381e470a6..3a795407d618 100644
--- a/packages/SystemUI/src/com/google/android/systemui/smartspace/KeyguardMediaViewController.java
+++ b/packages/SystemUI/src/com/google/android/systemui/smartspace/KeyguardMediaViewController.java
@@ -9,7 +9,7 @@ import android.os.UserHandle;
 import android.text.TextUtils;
 import android.view.View;
 
-import com.android.systemui.R;
+import com.android.systemui.customization.R;
 import com.android.systemui.broadcast.BroadcastDispatcher;
 import com.android.systemui.dagger.SysUISingleton;
 import com.android.systemui.plugins.BcSmartspaceDataPlugin;
@@ -126,7 +126,7 @@ public final class KeyguardMediaViewController {
         } else {
             charSequence = mediaMetadata.getText("android.media.metadata.TITLE");
             if (TextUtils.isEmpty(charSequence)) {
-                charSequence = context.getResources().getString(R.string.music_controls_no_title);
+                charSequence = context.getResources().getString(com.android.systemui.res.R.string.music_controls_no_title);
             }
         }
         CharSequence text =
-- 
2.49.0

